papas/pfalgo/pfreconstructor.py

```diff
@@ -344,6 +344,7 @@ def reconstruct_hcal(self, block, hcalid):
                         #again history is confusingbecause hcal is used to provide direction
                         #be better to make several smaller photons one per ecal?
                         parent_ids = [block.uniqueid] + [ecal.uniqueid for ecal in ecals]
+                        self.locked[hcal.uniqueid] = False # temp fix by alice
```

Comment:
Is that really fixing this issue? 
It seems to me that we would rather like to call reconstruct_cluster on hcal with some additional parameter that would say do_not_lock or something?
We will need to discuss.